.github/workflows/ci.yml

```diff
@@ -13,6 +13,8 @@ jobs:
       - uses: actions/checkout@v2
 
       - uses: purescript-contrib/setup-purescript@main
+        with:
```

Comment:
Need to update the Node version as well (below)

Reply:
Yup! Done.